packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFWebViewFlutterWKWebViewExternalAPI.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <Foundation/Foundation.h>
+#import <WebKit/WebKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+App and package facing native API provided by the `webview_flutter_wkwebview` plugin.
+
+This class follows the convention of breaking changes of the Dart API, which means that any
+changes to the class that are not backwards compatible will only be made with a major version
+change of the plugin.
+*/
+@interface FWFWebViewFlutterWKWebViewExternalAPI : NSObject
```

Comment:
Normally I'm not a fan of "classes" of just static functions in ObjC, but I guess it's probably for the best here to explicitly tie the API to the "external API" name.